packages/client/css/generic/button.css

```diff
@@ -0,0 +1,28 @@
+.Button {
+  padding: 10px 16px;
+  &:not(:first-child) { margin-left: 3px; }
+  &:not(:last-child) { margin-right: 3px; }
```

Comment:
Adding negatives (`:not`) tends to make things more confusing. Might it be better to say the following?

```css
margin-right: 6px;
&:last-child { margin-right: 0; }
```

This, I believe, would achieve the same effect: a `6px` gap between each `.Button`.